python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/system.j2

```diff
@@ -20,12 +20,20 @@
 | IPv6 | {{ system.control_plane.tcp_mss.ipv6 }} |
 {%         endif %}
 {%     endif %}
-{%     if system.control_plane.ipv4_access_groups is arista.avd.defined or system.control_plane.ipv6_access_groups is arista.avd.defined %}
+{%     if system.control_plane.ipv4_access_groups is arista.avd.defined or system.control_plane.ipv6_access_groups is arista.avd.defined or system.control_plane.ipv4_access_group_ingress_default is arista.avd.defined or system.control_plane.ipv6_access_group_ingress_default is arista.avd.defined %}
 
 #### Control-Plane Access-Groups
 
 | Protocol | VRF | Access-list |
 | -------- | --- | ------------|
+{#         IPv4 Access-groups Ingress Default #}
+{%         if system.control_plane.ipv4_access_group_ingress_default is arista.avd.defined %}
+| IPv4 Ingress default | All | {{ system.control_plane.ipv4_access_group_ingress_default }} |
+{%         endif %}
+{#         IPv6 Access-groups Ingress Default #}
+{%         if system.control_plane.ipv6_access_group_ingress_default is arista.avd.defined %}
+| IPv6 Ingress default | All | {{ system.control_plane.ipv6_access_group_ingress_default }} |
+{%         endif %}
```

Comment:
Just a "UX" suggestion - it may be better to move "IPv6 Ingress default" entry to the beginning of the "IPv6" items (so that all items of the same IP version are grouped together).
This may be useful in case user has a long list of IPv4 ACLs applied to different VRFs. In this case default IPv6 ACL would be separated from all other IPv6 ACLs by a long list of IPv4 ACL

Reply:
Good point, thanks.
Moved!